index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <link rel="stylesheet" href="stylesheets/css/all.min.css">
+    <title>Priscille's personal portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <div class="container">
+                <h1 class="logo"><a href="index.html">P.</a></h1>
+                <ul> 
+                    <li><a href="index.html">HOME</a></li>                  
+                    <li><a href="about.html">ABOUT</a></li>
+                    <li><a href="portfolio.html">PORTFOLIO</a></li>
+                    <li><a href="hobby-blog.html">BLOG</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="homepage">
+            <div class="img"></div>
+            <div class="beige"></div>
+        </div>
+        <div class="intro">
+                <h2>
+                    Hi, my name is Priscille <br> I am a full-stack web developer
+                </h2>
+                <a href="/" class="btn">View my work</a>
```

Comment:
The `/` lead to my own file system which was a little odd. You can use the `/` when your project is deployed, however, using `#` works just fine and will keep the user on the `index.html` page.

```suggestion
                <a href="#" class="btn">View my work</a>
```